pkg/controller/update/update.go

```diff
@@ -36,6 +36,11 @@ func (c *Controller) updateCommand(ctx context.Context, logE *logrus.Entry, para
 		if err != nil {
 			return fmt.Errorf("find a command: %w", err)
 		}
+
+		if findResult.Package == nil && findResult.ExePath != "" {
```

Comment:
If `findResult.Package` is nil, an error should be returned.
We don't need to check `findResult.ExePath`.

```suggestion
		if findResult.Package == nil {
```